widget/discrepancy-note/dn-widget.js

```diff
@@ -580,6 +583,15 @@ class Comment extends Widget {
 
     }
 
+    _scrollToview() {
+        const queryModal = this.linkedQuestion.querySelector( '.widget.or-comment-widget' );
+        if ( window.innerHeight - this.linkedQuestion.offsetHeight < queryModal.offsetHeight ) {
+            queryModal.scrollIntoView( { behavior: 'smooth', block: 'start', inline: 'nearest' } );
```

Comment:
Also nitpicky, but I would put the options for scrollIntoView in a `const` since they're used twice.